packages/iov-core/src/multichainsigner.spec.ts

```diff
@@ -132,4 +133,37 @@ describe("MultiChainSigner", () => {
       expect(acct.data[0].balance.length).toBe(1);
     });
   });
+
+  // transforms promise so resolved->rejected and rejected->resolved
+  const expectRejected = (prom: Promise<any>): Promise<boolean> =>
```

Comment:
Could you make this either return `Promise<void>` or `Promise<any>` (with the error from the catch handler returned)?

In the later case we could potentially check what kind of error we got

Reply:
okay, Promise<any> it is